x-pack/packages/kbn-elastic-assistant/impl/assistant_context/assistant_nav_link.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FC } from 'react';
+import React, { useCallback, useEffect, useMemo } from 'react';
+import ReactDOM from 'react-dom';
+import { createHtmlPortalNode, OutPortal, InPortal } from 'react-reverse-portal';
+import { ChromeNavControls } from '@kbn/core/public';
+import { EuiToolTip, EuiButton, EuiFlexGroup, EuiFlexItem, EuiButtonEmpty } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
```

Comment:
import `useKibana` from `security_solution/public/common/lib/kibana/kibana_react` so you'll have it with the right types

Reply:
Hey @semd, wouldn't this cause a circular dependency? Is it correct to import it like this: https://github.com/elastic/kibana/pull/203060/commits/94f48d013eecc5289d8b1443cf7efacc08cb87db#diff-c028c82d43280a6336e7e5b348f4f7e952cd60ea1fa7ef40594d5b0acb4faa9cR14

Reply:
Oh yeah, I did not realize we were in the assistant package 🤦 I was assuming this was _security_solution_ code. 
You are right, my previous suggestion is not correct. We should not load plugin _stuff_ from a package, we need to provide the `core.chrome` dep from the outside via `AssistantProvider`. 

I would add the entire `core` dependency to the assistant context, but to stay consistent with the current approach we should only add `chrome` to the `AssistantProvider`, and instead of using `useKibana ` we can retrieve it with:
```
const  { navControls, getChromeStyle$() } = useAssistantContext().chrome;
```

Sorry for the confusion